Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -100,7 +100,8 @@ private function createReferenceRelationTable(AbstractPlatform $platform): Table
         ]);
 
         return $table
-            ->setPrimaryKey(['name', 'position', 'nodeanchorpoint']);
+            ->setPrimaryKey(['name', 'position', 'nodeanchorpoint'])
```

Comment:
this is the first change after the release of Neos 9.0 to bring a schema adjustment which as you said must be executed via `./flow cr:setup`

we must explicitly denote that in the release notes as new users will not be aware of this additional set of migrations (aside from doctrine migrate)